.eslintrc.js

```diff
@@ -1,18 +1,21 @@
 module.exports = {
+  root: true,
```

Comment:
Would it be possible to “inherit” from these ESLint settings for the `safe-4337-passkeys` project specifically? This way we don’t need to pull in additional ESLint plugins for the packages in `modules/*` that don’t need them?

Reply:
Have added a config in the passkeys app file that extends this one.